A32-xds-circuit-breaking.md

```diff
@@ -93,16 +103,46 @@ The EDS LB policy will use this limit as the upper bound for the maximum number
 of in-flight requests it allows to send.
 
 The EDS policy's picker will enforce the configured limit. Each EDS LB policy
-will maintain a counter for the number of requests currently in flight to 
-the associated cluster. The counter will be incremented when a request
-is started on a subchannel and decremented when the request is complete. 
+will use a counter to aggregate the number of requests currently in flight to
+the associated cluster. The counter will be incremented when a request is
+started on a subchannel and decremented when the request is complete. 
 When the picker is going to start a call, it will check whether the current
 number of in-flight requests is lower than the configured limit. If so, the 
 picker will do the pick as usual, returning a subchannel. If not, it will 
 fail the call with status UNAVAILABLE. Such failed calls will not be retried,
 but they will be recorded to the `total_dropped_requests` counts in 
 cluster-level load reports and reported to the load reporting server.
 
+Recent changes to the load balancing policy hierarchy in gRPC split up the EDS
```

Comment:
This paragraph isn't needed.  We don't need to update this gRFC to reflect what comes after it, just to reflect the state when we implemented it.  Subsequent designs will always change it, and we don't go back and retrofit all previous gRFCs when that happens, so I'd rather not start giving people false hope that we'll do that. :)

Reply:
Deleted.